CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.19.5-1
+
+* Add `ngx_http_gzip_module` configuration.
```

Comment:
I left out compression as I assumed that we'd let the application or load balancer handle it. I'm not against adding this if neither are able to be configured with compression.

Reply:
Right. I'm ok with that.